app/src/main/AndroidManifest.xml

```diff
@@ -17,8 +17,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="true"
         android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/punchdrunker/hocho/blob/d64c32362bedf36cc0477fa783efd859d1e648c7/app/src/main/AndroidManifest.xml#L21"></span>Attribute <code>usesCleartextTraffic</code> is only used in API level 23 and higher (current min is 21)</td>
    </tr>
  </tbody>
</table>